src/clarity-angular/layout/tabs/tab-content.ts

```diff
@@ -14,22 +17,27 @@ let nbTabContentComponents: number = 0;
     template: `
         <ng-content></ng-content>
     `,
-    host: {
-        "[id]": "tabContentId",
-        "[attr.aria-labelledby]": "ariaLabelledBy",
-        "[attr.aria-hidden]": "!active",
-        "[attr.data-hidden]": "!active",
-        "role": "tabpanel"
-    }
+    host: {"role": "tabpanel"}
 })
-export class TabContent {
+export class TabContent implements OnDestroy {
     @ViewChild("tabContentProjectedRef") templateRef: TemplateRef<TabContent>;
 
     constructor(public ifActiveService: IfActiveService, @Inject(IF_ACTIVE_ID) public id: number,
-                private ariaService: AriaService) {
+                private ariaService: AriaService, private renderer: Renderer2, private el: ElementRef) {
         if (!this.tabContentId) {
             this.tabContentId = "clr-tab-content-" + (nbTabContentComponents++);
         }
+
+        this.setAttributes();
+        this.ifActiveService.currentChange.subscribe(() => {
```

Comment:
Looks like you're not adding this to the `subscriptions` array?

Reply:
Thanks. I have addressed the issue.